includes/class-wc-stripe.php

```
@@ -866,16 +867,38 @@ public function register_routes() {
+	 * requiring a gateway instance to already exist — used by {@see self::get_main_stripe_gateway()}
+	 * to pick which gateway class to instantiate.
+	 *
+	 * @return bool
+	 */
+	protected function is_optimized_checkout_enabled(): bool {
+		if ( ! WC_Stripe_Feature_Flags::is_oc_available() ) {
+			return false;
+		}
+		$settings = WC_Stripe_Helper::get_stripe_settings();
+		return isset( $settings['optimized_checkout_element'] ) && 'yes' === $settings['optimized_checkout_element'];
+	}
```

PHPMD (2.15.0)

[error] The method is_optimized_checkout_enabled is not named in camelCase. (undefined)

(CamelCaseMethodName)